src/components/pages/order/Main/Menu.js

```diff
@@ -1,23 +1,42 @@
-import { useState } from "react"
+import { useContext } from "react"
 import styled from "styled-components"
-import { fakeMenu2 } from "../../../../data/fakeMenu"
 import Card from "../../../reusable-ui/Card"
 import { formatPrice } from "../../../../utils/maths"
 import { theme } from "../../../../theme"
+import MenuContext from "../../../../context/MenuContext"
+import AdminContext from "../../../../context/AdminContext"
+import EmptyMenuPage from "./EmptyMenuPage"
 
 export default function Menu() {
-	const [menu, setMenu] = useState(fakeMenu2)
+	const { menu, setMenu } = useContext(MenuContext)
+	const { isAdminMode } = useContext(AdminContext)
+
+	const comingSoonImageUrl = "/images/coming-soon.png"
+
+	const handleClick = (id) => {
+		const selectedItem = menu.find((item) => item.id === id)
+
+		const newMenu = [...menu]
+
+		newMenu.splice(newMenu.indexOf(selectedItem), 1)
+
+		setMenu(newMenu)
+	}
 
 	return (
 		<MenuStyled>
-			{menu.map(({ id, title, imageSource, price }) => (
-				<Card
-					key={id}
-					title={title}
-					imageSource={imageSource}
-					leftDescription={formatPrice(price)}
-				/>
-			))}
+			{menu.length !== 0
+				? menu.map(({ id, title, imageSource, price }) => (
+						<Card
+							key={id}
+							title={title}
+							imageSource={imageSource ? imageSource : comingSoonImageUrl}
```

Comment:
Oui, très bien ! 💯

Reply:
```suggestion
							imageSource={imageSource ? imageSource : defaultImage}
```

Tu peux aller encore plus loin et donner un nom explicite à ta variable